### DISCUSSION TITLE
You can refurbish rides that are selected for preservation

### DISCUSSION THREAD

**Discussion by MarcelVos96** [OpenRCT2/OpenRCT2#22731] [Feature requests]
### Operating System

Windows 10

### OpenRCT2 build

v0.4.14

### Base game

RollerCoaster Tycoon 2

### Area(s) with this issue?

This is a development issue

### Describe the issue

Ride preservation makes it so that you cannot alter or delete the ride. However, right now you can still refurbish the rides, removing the "old historic rides" aspect of this which you often see in the scenarios.

Solution: Make the refurbish button give the same error message as the contruction button gives when a ride is selected for preservation.

### Steps to reproduce

Open Haunted Harbor and try to delete and refurbish the wooden coaster, which is selected for preservation.

**Comment by 733737** [comment]
refurbishing old rides is a form of preservation.

**Comment by Gymnasiast** [comment]
I don’t think this is a bug, so I moved it to Discussions.

If this is game-breaking, I’d like to hear exactly why.

**Reply by MarcelVos96** [reply]
Yeah that's fair, I reported it as a bug originally because the creator of the refurbish option said it was an oversight, but I can see there's multiple perspectives on it now.

**Comment by MASHzj** [comment]
I can see an argument in both ways, it's mainly depending on the kind of scenario. For example in Haunted Harbour it's better that the very old rides cannot be refurbished. While in finish 5 coasters scenarios, it's nice to be able to refurbish them if you're playing for a long time.

**Comment by Ling1375401** [comment]
Sorry to give this a kick, but I was doing research for a different post and came across it. Allowing this in several scenarios fundamentally changes their character IMO; basically any park value scenario _or_ any fix-up-an-old-park scenario. I think the refurbish feature is an issue on its own (vastly too easy and too cheap for what it does, it is basically just the cheat version of itself but with a requirement to close the ride for a second) but separate from that it should not apply to these protected/intentionally old rides. You can trivially double or triple the park value of these fix-up-old-park scenarios instantly and give yourself massive cash flow where you were obviously intended to struggle at first under the weight of a huge loan and very low ride income.

**Reply by Ling1375401** [reply]
Maybe allowing refurbishment could be something that could be baked in to a scenario. Just completely preclude it from any pre-existing scenario (at least RCT1 scenarios? there are no old-park-fixer-upper scenarios in RCT2 vanilla), but allow map makers to turn it on or off like they do Harder Guest Gen, Harder Park Rating, etc. for future ones. If you're playing for a really long time money should be no issue and there are other ways to prop up ride reliability.
